lib/controllers/root-controller.js

```diff
@@ -69,21 +73,16 @@ export default class RootController extends React.Component {
     super(props, context);
     autobind(
       this,
-      'installReactDevTools', 'clearGithubToken', 'initializeRepo', 'showOpenIssueishDialog',
-      'showWaterfallDiagnostics', 'showCacheDiagnostics', 'acceptClone', 'cancelClone', 'acceptInit', 'cancelInit',
-      'acceptOpenIssueish', 'cancelOpenIssueish', 'destroyFilePatchPaneItems',
-      'destroyEmptyFilePatchPaneItems', 'openCloneDialog', 'quietlySelectItem', 'viewUnstagedChangesForCurrentFile',
+      'installReactDevTools', 'clearGithubToken',
+      'showWaterfallDiagnostics', 'showCacheDiagnostics',
+      'destroyFilePatchPaneItems', 'destroyEmptyFilePatchPaneItems',
+      'quietlySelectItem', 'viewUnstagedChangesForCurrentFile',
       'viewStagedChangesForCurrentFile', 'openFiles', 'getUnsavedFiles', 'ensureNoUnsavedFiles',
       'discardWorkDirChangesForPaths', 'discardLines', 'undoLastDiscard', 'refreshResolutionProgress',
     );
 
     this.state = {
-      cloneDialogActive: false,
-      cloneDialogInProgress: false,
-      initDialogActive: false,
-      initDialogPath: null,
-      initDialogResolve: null,
-      credentialDialogQuery: null,
+      dialogRequest: dialogRequests.null,
```

Comment:
One of the big reasons I wanted to do this was to make this state less of a "blob of everything."